README.md

```diff
@@ -1,62 +1,397 @@
-# Styled components
+<a href="https://github.com/styled-components/styled-components">
+  <img alt="styled-components" src="./docs/assets/logo.png" height="150px" />
+</a>
+<br />
 
-**This is a work in progress** based off of [this demo](https://github.com/geelen/css-components-demo).
+Visual primitives for the component age. Use the best bits of ES6 and CSS to style your apps without stress 💅
 
-### Usage
+```
+npm install --save styled-components
+```
+
+[![Travis Build Status](https://travis-ci.org/styled-components/styled-components.svg?branch=master)](https://travis-ci.org/styled-components/styled-components) [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/gruntjs/grunt?branch=master&svg=true)](https://ci.appveyor.com/project/mxstbr/styled-components) [![Supported by Thinkmill](https://thinkmill.github.io/badge/heart.svg)](http://thinkmill.com.au/?utm_source=github&utm_medium=badge&utm_campaign=styled-components)
```

Comment:
Yeah fair enough. The latest version number is always on the releases tab anyway.